pkg/api/http_server.go

```diff
@@ -126,7 +126,8 @@ type HTTPServer struct {
 	RouteRegister                routing.RouteRegister
 	RenderService                rendering.Service
 	Cfg                          *setting.Cfg
-	Features                     *featuremgmt.FeatureManager
+	Features                     featuremgmt.FeatureToggles
+	featureManager               *featuremgmt.FeatureManager
```

Comment:
^^ this is the most awkward change.  Really points out that this file is the messy jumble of everything 🤮